app/controllers/songs_controller.rb

```diff
@@ -17,9 +17,10 @@ def index
         songs = songs.search_by_keywords(search_value) if search_value.present?
         songs = songs.where(key: params[:key]) if params[:key].present?
         songs = songs.where(tempo: params[:tempo]) if params[:tempo].present?
+        songs = songs.where(category: params[:category]) if params[:category].present?
         # get number of matching songs post filtering
         records_filtered = songs.count
-        songs = songs.select('id, artist, tempo, key, name, chord_sheet, spotify_uri')
+        songs = songs.select('id, artist, tempo, key, name, chord_sheet, spotify_uri, category')
```

Comment:
We're hopefully eventually going to deprecate the server side rendering and use the API. Can you duplicate this change in the api/songs_controller please?